Framework/PythonInterface/mantid/kernel/funcinspect.py

```diff
@@ -106,18 +106,17 @@ def decompile(code_object):
     instructions = []
     n = len(code)
     i = 0
-    e = 0
     while i < n:
         i_offset = i
         i_opcode = opcode_from_bytecode(code[i])
         i = i + 1
         if i_opcode >= opcode.HAVE_ARGUMENT:
-            i_argument = opcode_from_bytecode(code[i]) + (opcode_from_bytecode(code[i+1]) << (4*2)) + e
-            i = i + 2
-            if i_opcode == opcode.EXTENDED_ARG:
-                e = iarg << 16
+            if sys.version_info < (3, 6): # from 3.6 bytecode is now 16-bit not 8-bit
```

Comment:
The changes make sense. Is it worth caching the version check at the module level to save re-evaluating it?